### PR TITLE
fix: alert metric with dependencies repair

### PR DESCRIPTION
**Pull request by hackerstanislav**
JIRA: F1-805

---

> [!IMPORTANT]
> Please, **don't forget to run `rush change`** for the commits that introduce **new features** 🙏

---

Refer to [documentation](https://github.com/gooddata/gooddata-ui-sdk/blob/master/dev_docs/continuous_integration.md) to see how to run checks and tests in the pull request. This is the list of the most used commands:

```
extended test - backstop
```

```
extended test - tiger-cypress - integrated
extended test - tiger-cypress - isolated
extended test - tiger-cypress - record
```

### CODE REVIEW

File: libs/sdk-backend-tiger/src/convertors/fromBackend/AutomationConverter.ts [hunk 1]
```diff
@@ -159,8 +159,8 @@ const convertAlert = (
                 operator: comparison.operator as IAlertComparisonOperator,
                 left: {
                     id: comparison.left.localIdentifier,
-                    title: comparison.left.title,
-                    format: comparison.left.format,
+                    title: comparison.left.title ?? undefined,
```
Comment (kandl) [inline libs/sdk-backend-tiger/src/convertors/fromBackend/AutomationConverter.ts:162]: Why is this needed?

**Review by kandl — APPROVED** [review]
LGTM

File: libs/sdk-backend-tiger/src/convertors/fromBackend/AutomationConverter.ts (hunk 1 above)
Comment (hackerstanislav) [reply libs/sdk-backend-tiger/src/convertors/fromBackend/AutomationConverter.ts:162]: Api now accept null values here but we want only undefined ....
